examples/LLM/llama2/chat_app/Readme.md

```diff
@@ -9,6 +9,33 @@ We are using [llama-cpp-python](https://github.com/abetlen/llama-cpp-python) in
 You can run this example on your laptop to understand how to use TorchServe
 
 
+## Quick Start Guide
+
+To get started with TorchServe, you need to run the following
+
+```
+# 1: Set HF Token as Env variable
+export HUGGINGFACE_TOKEN=<Token> # get this from your HuggingFace account
+
+# 2: Build TorchServe Image for Serving llama2-7b model with 4-bit quantization
+./examples/llm/llama2/chat_app/docker/build_image.sh meta-llama/Llama-2-7b-chat-hf
+
+# 3: Launch the streamlit app for server & client
```

Comment:
I know it's not exactly what you might have in mind but I was thinking this would open a terminal based CLI

Reply:
I was thinking about how to cover various scenarios..
So, my goal is to do a 3 part solution

1) chatbot quickstart with streamlit -> Because chatbots are popular
2) TS multi model app to show TS' full capability - Use this to create video series
3) quick start script for common use-cases with curl command -> This can be the getting started guide.